app/assets/javascripts/users/settings/teams/invite_users_modal.js

```diff
@@ -123,23 +137,31 @@
           dataType: 'json',
           data: data,
           success: function(data) {
-            animateSpinner(modalDialog, false);
-            stepForm.hide();
-            stepResultsDiv.html(data.html);
-            stepResults.show();
-            // Add 'data-invited="true"' status to modal element
-            modal.attr('data-invited', 'true');
+            if (data.recaptcha_error) {
+              animateSpinner(modalDialog, false);
+              recaptchaErrorMsgDiv.removeClass('hidden');
+              recaptchaErrorText.text(data.recaptcha_error);
+              reloadRecaptchaIfExists();
+            } else {
```

Comment:
In the ideal world this should be handled by error callback. For this controller should return an HTTP error code (for example 422)

Reply:
Agree 👍